BackEnd/Core/sailfish-common/src/main/java/com/exactpro/sf/common/impl/messages/all/configuration/sample/Message.java

```diff
@@ -0,0 +1,81 @@
+//
+// This file was generated by the JavaTM Architecture for XML Binding(JAXB) Reference Implementation, v2.2.7 
+// See <a href="http://java.sun.com/xml/jaxb">http://java.sun.com/xml/jaxb</a> 
+// Any modifications to this file will be lost upon recompilation of the source schema. 
+// Generated on: 2019.12.12 at 02:34:49 PM GMT+04:00 
+//
+
+
+package com.exactpro.sf.common.impl.messages.all.configuration.sample;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlType;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.exactpro.sf.common.impl.messages.all.configuration.sample.iface.IField;
+import com.exactpro.sf.common.impl.messages.all.configuration.sample.iface.IMessage;
+
+/**
+ * <p>Java class for Message complex type.
+ * 
+ * <p>The following schema fragment specifies the expected content contained within this class.
+ * 
+ * <pre>
+ * &lt;complexType name="Message">
+ *   &lt;complexContent>
+ *     &lt;extension base="{http://exactprosystems.com/dictionary}Field">
+ *       &lt;sequence>
+ *         &lt;element name="field" type="{http://exactprosystems.com/dictionary}Field" maxOccurs="unbounded" minOccurs="0"/>
+ *       &lt;/sequence>
+ *     &lt;/extension>
+ *   &lt;/complexContent>
+ * &lt;/complexType>
+ * </pre>
+ * 
+ * 
+ */
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "Message", propOrder = {
+    "fields"
+})
+public class Message extends Field implements IMessage, Serializable
+{
+
+    private final static long serialVersionUID = -1L;
+    @XmlElement(name = "field", type = Message.class)
```

Comment:
`type = Message.class` should be replaced by Field.class.
Or we can use a more specific type for the list like `List<Field>`